fix-point/fix-point.cabal

```diff
@@ -0,0 +1,61 @@
+-- This file has been generated from package.yaml by hpack version 0.28.2.
+--
+-- see: https://github.com/sol/hpack
+--
+-- hash: 4a50b2893a321b706d4c697fb5f55fe89bbbfe0c5ceb8012711f9d16879ae520
+
+name:           fix-point
+version:        0.1.0.0
+description:    Please see the README on GitHub at <https://github.com/githubuser/fix-point#readme>
+homepage:       https://github.com/githubuser/fix-point#readme
+bug-reports:    https://github.com/githubuser/fix-point/issues
+author:         Author name here
+maintainer:     example@example.com
+copyright:      2018 Author name here
+license:        BSD3
+license-file:   LICENSE
```

Comment:
Did you try to build this package? Is it buildable?

Reply:
Package is buildable and executable without error.